apps/intake/tests/utils/in-person/FillingInfo.ts

```diff
@@ -87,6 +80,28 @@ export class FillingInfo {
     return { firstName, lastName, birthSex, email, reasonForVisit, enteredReason };
   }
 
+  async fillNewPatientInfoSmoke() {
+    const firstName = TEST_PATIENT_FIRST_NAME;
+    const lastName = TEST_PATIENT_LAST_NAME;
+    // cspell:disable-next ykulik
```

Comment:
you should move this line to `packages/test-utils/lib/e2e/data.ts` so cspell works correctly